neclib/devices/signal_generator/fsw0010.py

```diff
@@ -0,0 +1,107 @@
+import time
+import ogameasure
+from ... import config
+from .signal_generator_base import SignalGenerator
+
+
+class FSW0010(SignalGenerator):
+
+    Manufacturer: str = "PhaseMatrix"
+    Model = "FSW0010"
+
+    def __init__(self):
+        self.busy = False
+        com = ogameasure.ethernet(config.rx_fsw0010_host, config.rx_fsw0010_port)
+        self.sg = ogameasure.Phasematrix.FSW0010(com)
+        self.sg.use_external_reference_source()
+
+    def set_freq(self, freq_GHz):
+        if self.busy is True:
+            self.busy = False
+            self.sg.freq_set(freq_GHz)
+            time.sleep(1)
+            self.busy = True
```

Comment:
`self.busy` が `True` の場合は「他の処理を実行中で新しい処理は受け付けられない」状態です。
そのためできれば busy でなくなるまで待機し、その後に通信 (`self.sg.freq_set`) を行うように変更してほしいです。

Reply:
@KaoruNishikawa 
```suggestion
        while self.busy is True:
            time.sleep(1)
```
if文ではなくwhile文でこのように書いたらいいでしょうか。

Reply:
@yamatomatsuzuki 
良いと思います！